### PR TITLE
Uploaded magnetopause crossing detection chapter

### PR DESCRIPTION
**Pull request by colinrsmall**

### CODE REVIEW

**Comment by mbobra** [comment]
Thank you so much for your PR, @colinrsmall! We will review and merge sometime in the next couple weeks.

**Review by jmason86 — COMMENTED** [review]
Looks good on first quick look. Special thanks for filling in the details in the table of contents; that makes our job much easier. Extra special thanks for including your dependencies in the environment file with version specified. We've run into trouble with that on some of the older chapters and are now working through all of that after the fact. Much easier to specify version upon initial commit, so thanks!

**Comment by colinrsmall** [comment]
Of course! Glad it's all in order. I just took a look through the notebook again just now and noticed that one of the cells isn't displaying properly. I'll fix that and re-commit.

**Comment by jmason86** [comment]
@mbobra I'm not able to view the notebook -- it won't load in github in any browser I've tried and it also won't load with [nbviewer](https://nbviewer.jupyter.org/). We might have to merge and render through our process and check it out live. That's probably an issue we should create, though the solution will probably just be that same plan we have to make a test area.

**Comment by colinrsmall** [comment]
@jmason86 That's one of the issues I fixed. Can you try viewing it now?

**Comment by jmason86** [comment]
Yep, it works now in both github and nbviewer; thanks! Reviewing now. 

> @jmason86 That's one of the issues I fixed. Can you try viewing it now?
> 
> Edit: The cell I needed to fix is fixed now too. It should be good to go.

**Comment by jmason86** [comment]
Notebook review: 

* Excellent overall! 
* Great introduction. 
* Thanks for that second authors' note especially. 
* Solid use of Zenodo for accessing the data files.
* Lots of great explanatory text
* Clear variable names

- [ ] Typo in introduction "... the development steps and hyperparamteres" should be "hyperparameters" - Can't believe I caught that
- [ ] The first "Authors' note" is repeated verbatim
- [ ] If there are any additional steps for enabling a GPU in Google Collab, could you provide a link to instructions for that somewhere on the web in your second Authors' note?
- [ ] I know this is nitpicky, but could you increase the font size on your axis labels in plots? Particularly the Model Performance Visualization and Model Validation plots (big fig size has the effect of shrinking the apparent font size)
- [ ] Could you include just a little bit at the very end to wrap up: conclusions, summary, interpretations, whatever

**Comment by colinrsmall** [comment]
>* [ ]  Typo in introduction "... the development steps and hyperparamteres" should be "hyperparameters" - Can't believe I caught that

Good catch, thanks!

> * [ ]  The first "Authors' note" is repeated verbatim

Fixed.

* [ ]  If there are any additional steps for enabling a GPU in Google Collab, could you provide a link to instructions for that somewhere on the web in your second Authors' note?

I couldn't find any simple instructions directly from Google, so I wrote out the instructions in the authors' note. I can move the instructions to under the cell where a user may see TF Keras complain about not having a GPU accessible if you think that'll be easier for readers.

* [ ]  I know this is nitpicky, but could you increase the font size on your axis labels in plots? Particularly the Model Performance Visualization and Model Validation plots (big fig size has the effect of shrinking the apparent font size)

The font's now size 18. Let me know if you want it bigger still.

* [ ]  Could you include just a little bit at the very end to wrap up: conclusions, summary, interpretations, whatever

Added.

**Comment by jmason86** [comment]
Looks great! No need to move the instructions about the GPU. We'll wait for @mbobra to give it a once over before merging, but that's it from me!

**Review by mbobra — APPROVED** [review]

**Comment by mbobra** [comment]
[Chapter 8](https://helioml.org/08/Automated_Detection_Of_Magnetopause_Crossings.html) is live and looks super awesome! 🎉🎉🎉

Thank you for your contribution @colinrsmall @argallmr @marekpetrik!
